### ISSUE TITLE
Default for avconv setting on 18.04 wrong

### ISSUE THREAD

**Issue by wxmerkt** [Gepetto/gepetto-viewer#106]
Relates to #92 and #94.

#94 added the ability to configure `avconv` to be `ffmpeg` or whatever we want. However, in the robotpkg distributed viewer, it still points to `avconv` on 18.04 by default - it is now `ffmpeg` again. Can this be fixed in the distributed version?

**Comment by julesser** [comment]
This seems to be somehow related to an error I currently receive. When trying to record a movie of the crocoddyl example _bipedal_walking.py_ in the gepetto viewer, it fails with the following log:

```
Failed to start avconv. Either it is missing, or you may have insufficient permissions to invoke it. 

You can manually run avconv -r 25 -i /tmp/gepetto-gui/record/img_0_%d.jpeg -vf scale=trunc(iw/2)*2:trunc(ih/2)*2 -r 25 -vcodec libx264 /home/###hidden###/Videos/bipedalRecording.mp4
```

**Comment by jmirabel** [comment]
@nim65s there are several possible workaround:
- use cmake to know what the default should be. Then, the line to change is https://github.com/Gepetto/gepetto-viewer/blob/0169d4b4da6c22fdc6354913b4a9fc1a8f768087/src/gui/settings.cc#L49
- otherwise, we could look for both and see if any is found.

**Comment by nim65s** [comment]
Thanks for the heads-up, I had forgotten this.

**Comment by jmirabel** [comment]
Fixed by #119
